### ISSUE TITLE
PEP.MS2Quantity missing error

### ISSUE THREAD

**Issue by ethangithub1017** [liuzan-info/PhosMap#38]
Hi,

I have encountered trouble in uploading the Spectronaut result. I have included the PEP.MS2Quantity in my result but PhosMap keeps saying that the PEP.MS2Quantity data is missing. Can I contact you via email so I can share my dataset with you to have a look?

Thanks
Ethan

**Comment by liuzan-info** [comment]
Certainly, I would be happy to assist you. Please feel free to reach out to me at [liuzan_info@163.com](mailto:liuzan_info@163.com).
